components/dashboard/public/index.html

```diff
@@ -17,14 +17,14 @@
     <link rel="icon" type="image/svg+xml" href="/images/gitpod.svg" sizes="any"/>
     <link rel="stylesheet" href="/styles.css"/>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat" />
-    <title>Gitpod - Online IDE for GitHub and GitLab</title>
-    <meta name="description" content="Describe your dev environment as code and get fully prebuilt, ready-to-code development environments for any GitLab, GitHub, and Bitbucket project.">
+    <title>Gitpod - Dev environments built for the cloud</title>
```

Comment:
Ah, sorry, this one should be an endash too:

```suggestion
    <title>Gitpod – Dev environments built for the cloud</title>
```